src/app.js

```diff
@@ -1,74 +1,72 @@
 const express = require('express')
 const cors = require('cors')
 const helmet = require('helmet')
-const passport = require("passport")
+const passport = require('passport')
 const session = require('express-session')
-const path = require("path")
+const path = require('path')
 
-const { cloudinary } = require('./utils/helpers')
-const userRoutes = require("../src/modules/user/user.routes")
-const productRoute = require('./modules/product.routes')
-const profileRoute = require('./modules/profile.route')
-const categoryRoutes = require('./routes/category.route')
+const userRoutes = require('./modules/user/user.route')
+const productRoutes = require('./modules/product/product.route')
+const profileRoutes = require('./modules/profile/profile.route')
+const categoryRoutes = require('./modules/category/category.route')
 
 const app = express()
 
-app.set("view engine", "ejs")
-app.set("views", path.join(__dirname, "views"))
+app.set('view engine', 'ejs')
+app.set('views', path.join(__dirname, 'views'))
 
 app.use(express.urlencoded({ extended: true }))
     .use(express.json({ limit: '50kb' }))
     .use(cors())
     .use(helmet())
-    .use('/product', productRoute)
-    .use('*', cloudinary)
```

Comment:
you removed the cloudinary middleware  ( app.use("*", cloudinary) ) in app.js file. you can delete the model and route folder, since we are not using them again.  Also in the modules folder delete the product files that are not inside the product folder.

Reply:
The app.js file has been fixed.